hoodie-common/src/main/java/com/uber/hoodie/common/table/HoodieTableConfig.java

```diff
@@ -119,6 +127,18 @@ public HoodieTableType getTableType() {
     }
 
     /**
+     * Read the merge strategy for HoodieRecords from the table properties
+     *
+     * @return
+     */
+    public String getPayloadClass() {
+        if (props.containsKey(HOODIE_PAYLOAD_CLASS_PROP_NAME)) {
+            return props.getProperty(HOODIE_PAYLOAD_CLASS_PROP_NAME);
```

Comment:
is nt there getProperty(k, defaultVal), which can make this a single line

Reply:
Cool, incorporated.